py3-charset-normalizer.yaml

```diff
@@ -35,7 +35,7 @@ pipeline:
     with:
       repository: https://github.com/jawah/charset_normalizer.git
       tag: ${{package.version}}
-      expected-commit: 1e10d06e26bacaab3b513e601037889d00ae54ad
+      expected-commit: 9720055dde3c146db76c7195f3c474df6212191e
```

Comment:
I see this commit is one newer, but there are also some additional changes. While those shouldn't affect us, it doens't look like they actually tagged a release or cut a release either. Should we hold off on a release till they have officially done so?

Reply:
Oh interesting - yea for sure, agreed looks like they changed their minds, will close this one and automation will re-pick up when they issue an update, thanks :)